PendingReleaseNotes.md

```diff
@@ -10,8 +10,10 @@ v1.5...
 
 ### Ceph
 
+- Ceph mons require an odd number for a healthy quorum. An even number of mons is now disallowed.
```

Comment:
Just to make sure, is the Rook Ceph Cluster CRD documentation and / or pre-requisites for Ceph on this topic?

Reply:
Good point, i'll make sure it's clear in the examples and docs